chainstate/test-framework/src/framework_builder.rs

```diff
@@ -58,7 +58,7 @@ pub struct TestFrameworkBuilder {
 
 impl TestFrameworkBuilder {
     /// Constructs a builder instance with values appropriate for most of the tests.
-    pub fn new(_rng: &mut (impl Rng + CryptoRng)) -> Self {
+    pub fn new(rng: &mut (impl Rng + CryptoRng)) -> Self {
```

Comment:
I'd say the doc comment here should mention that `tx_index_enable` is determined randomly by default

Reply:
I don't think this is helpful because this will change in the future as more things will become random. Docs tend to not be updated.